### DISCUSSION TITLE
Drop Helm v2 support

### DISCUSSION THREAD

**Discussion by yxxhero** [helmfile/helmfile#390] [Ideas]
drop helm v2 support in helmfile v1.0

**Comment by mumoshu** [comment]
@yxxhero AFAIK Helm v2 has been deprecated since 2020. Can we probably drop the support for helm v2 before helmfile v1, without fears of breaking anything, as people shouldn't be using helm v2 anymore? 🤔

**Reply by mumoshu** [reply]
cc/ @xiaomudk @itscaro

**Comment by xiaomudk** [comment]
Good idea! This can reduce a lot of maintenance work. For users using helm v2, we can guide users to use the old version of helmfile in the readme.

**Comment by xiaomudk** [comment]
I think I can give it a try.
